src/components/GlobalStyle.js

```diff
@@ -23,6 +22,26 @@ const GlobalStyle = createGlobalStyle`
     line-height: inherit;
   }
 
+  h1, h2, h3 {
+    font-weight: normal;
+    color: ${(props) => props.color};
+    margin: 0;
+  }
+
+  h1 {
```

Comment:
Should we create its own component like we do in stellar.org, @vcarl

Reply:
Yeah, definitely need components for all these. Don't want to rely on DOM-specific details since this will be consumed by react native as well.

Reply:
Sounds good. Question - the design of Stellar.org and react native is very different. Will Web components be used by react native?

Reply:
Yeah, since this will be consumed by React Native as well we need components for everything.

Reply:
I'm not exactly sure how the interplay will work out, but if nothing else it'd be nice to have a consistent API across web and native. e.g. always using `<Text>`, <OrderedList>` etc.

Reply:
We don't have immediate plans to use React Native, but I think it's just a better API to have components for headers, rather than making a global overwrite. It just so happens that doing that also removes a potential inconsistency with React Native, so that seems like an added plus to me.

Reply:
Sounds good!